### ISSUE TITLE
Templates: Do they all need to be templates

### ISSUE THREAD

**Issue by rcass** [redacademy/tamwood-spring-2018#23]
So I can see your group has created a lot of page templates, where the user can select.
![screen shot 2018-06-21 at 2 17 40 pm](https://user-images.githubusercontent.com/5028555/41746216-685bee6c-755e-11e8-9c0a-57d90704cba8.png)

However, I am not sure if all of these should be templates the user has access to turning on and off. Optional page templates are really great for having different templates for different design patterns (1 column vs 2 column page). If you know that one page will ALWAYS have a certain design and want to make sure the user always goes to that page, I would just create a custom template instead.

Basically, the php name would look something like this.
> page-{slug}.php — If no custom template has been assigned, WordPress looks for and uses a specialized template that contains the page’s slug.

And you would need to remove the comment at the top of the page. 

Here is the documentation for more detailed info: https://developer.wordpress.org/themes/template-files-section/page-template-files/#uses-for-page-templates
